src/custom/utils/environments.ts

```diff
@@ -22,4 +22,20 @@ export function checkEnvironment(host: string) {
 
 const { isDev, isStaging, isProd, isEns, isPreStaging } = checkEnvironment(window.location.host)
 
+export const environmentName = (function () {
+  if (isProd) {
+    return 'production'
+  } else if (isPreStaging) {
+    return 'barn'
+  } else if (isEns) {
+    return 'ens'
+  } else if (isStaging) {
+    return 'staging'
+  } else if (isDev) {
+    return 'development'
+  } else {
+    return undefined
```

Comment:
What about

```suggestion
    return 'unknown'
```

Reply:
I would prefer to define it as undefined, cause it is what it is, since it's not an environment :) (i just don't know what it is)

In this case, its only used to pass the data to sentry environment, which accepts undefined, and has the meaning of unspecified, exactly what i want to do.